admin-dev/themes/new-theme/js/pages/product/services/categories.ts

```diff
@@ -23,12 +23,14 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+/* eslint-disable max-len */
+
 import Router from '@components/router';
 
 const router = new Router();
 const {$} = window;
 
-export const getCategories = async () => $.get(router.generate('admin_categories_get_categories_tree'));
+export const getCategories = async (): Promise<Array<TreeCategory>> => $.get(router.generate('admin_categories_get_categories_tree'));
```

Comment:
Is it worth keeping this on a single line and adding a disable comment?

Reply:
It's already a single line?

Reply:
My bad I guess Github display confused me 😅